### PR TITLE
TRUNK-5058 : Extract private method in ObsServiceImpl.saveExistingObs

### PR DESCRIPTION
**Pull request by PKatGITHUB**
## Related Issue

see https://issues.openmrs.org/browse/TRUNK-5058

## Checklist:

- [x] My pull request only contains one single commit.
- [x] My pull request is based on the latest master branch
  `git pull --rebase upstream master`.
- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.
- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by PKatGITHUB** [comment]
@dkayiwa  please have a look at it.

**Comment by PKatGITHUB** [comment]
@dkayiwa is this one fine?

File: api/src/main/java/org/openmrs/api/impl/ObsServiceImpl.java [hunk 1]
```diff
@@ -173,6 +165,18 @@ private Obs saveExistingObs(Obs obs, String changeMessage) {
 		return newObs;
 
 	}
+        
+        private void unsetVoidedAndCreationProperties(Obs newObs,Obs obs) {
+                // unset any voided properties on the new obs
```
Comment (teleivo) [inline api/src/main/java/org/openmrs/api/impl/ObsServiceImpl.java]: @PKatGITHUB please remove the comments in this method. the method name is now long and meaningful enough that the comments are not needed.

**Comment by teleivo** [comment]
@dkayiwa added a comment. if thats done then it can be merged

**Comment by PKatGITHUB** [comment]
@teleivo  ok i am doing it

**Pull request merged**